.swiftformat

```diff
@@ -1,3 +1,7 @@
+# file options
+
+--exclude Examples/ExampleMultiProjectIntegration/ExampleMultiProjectIntegration/SafeDI.swift
```

Comment:
The linter does not approve of the code-gen. Go figure.

Reply:
What specifically doesn't it like? Worth it to fix?

Reply:
It's my own swift format config that I'm fighting. Not worth fixing (at least not in the short term) – this generated code isn't meant to be pretty as much as it's meant to be easy to write + maintain.

Biggest delta is that my format config wants `@MainActor` on its own line rather than on the same line as `func`.